classes/WP_CLI/ActionScheduler_WPCLI_Scheduler_command.php

```diff
@@ -5,6 +5,37 @@
  */
 class ActionScheduler_WPCLI_Scheduler_command extends WP_CLI_Command {
 
+	/**
+	 * Force tables schema creation for Action Scheduler
+	 *
+	 * ## OPTIONS
+	 *
+	 * @param array $args Positional arguments.
+	 * @param array $assoc_args Keyed arguments.
+	 * @throws \WP_CLI\ExitException When an error occurs.
```

Comment:
```suggestion
```

I don't think this exception can be thrown by this method (the only call into WP CLI is to `WP_CLI::success()` which won't trigger this).